csv_to_vanilla-i18n/csv_or_sheet_to_vanilla_i18.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python3
```

Comment:
Please remove this file as the changes have been made in the original py script.